sources/ghostdog/src/main.rs

```diff
@@ -50,7 +50,7 @@ fn run() -> Result<()> {
 }
```

Comment:
I presume you made this public to access for fuzzing, maybe we can if-gate this so to not modify the main functionality?

Reply:
this doesn't functionally change anything about the public api of the crate. Ghostdog is a binary crate that just produces an executable. For fuzzing, cargo fuzz relies on importing a library to fuzz an API. That's why I created the _src/lib.rs_ file (and it is only active under conditional compilation!). In short, internal functions that become `pub` have no effect on executable programs, but it is necessary to plumb it through the library interface.